docs/cse/rules/cse-rules-syntax.md

```diff
@@ -1162,6 +1162,19 @@ Filters results based on the value of a boolean expression.  
 
 `| where jsonArrayContains(field, “vuln_scanner”)`  
  
+## in
```

Comment:
It looks like this page is in alphabetical order, so you'll need to move "in" to the right spot

Reply:
@kimsauce: Thank you so much for pointing out this. I have fixed it.

Reply:
thank you!